lib/file.js

```diff
@@ -34,7 +34,7 @@ class File {
       return false
     }
 
-    return !!this.name.match(/\.(exe)$/)
+    return !!this.name.match(/\.(lnk)$/)
```

Comment:
I would not think so, the directories involved do not normally contain exe's.

Also it only takes a moment to create a shortcut for an exe on your desktop so it can be launched, users are used to doing this, also almost all installers create a shortcut by default.

In Windows users almost always launch via a shortcut.